packages/x-components/src/design-system-deprecated/components/picture/default.scss

```
@@ -1,4 +1,6 @@
 .x-picture {
+  overflow: hidden;
```

Set here the `display: block`, it was already set before and it's been set in the new XDS.

```suggestion
  overflow: hidden;
  display: block;
```